templates/mongodb-atlas.base-template.yaml

```diff
@@ -0,0 +1,329 @@
+      - DatabaseUserRoleDatabaseName
+      - VPCCIDR
+      - PrivateSubnetCIDR
+      - AvailabilityZone
+      - VPCRegion
+    ParameterLabels:
+      PublicKey:
+        default: MongoDB Atlas API PublicKey
+      PrivateKey:
+        default: MongoDB Atlas API PrivateKey
+      OrgId:
+        default: MongoDB Atlas API OrgId
+      ProjectName:
+        default: Name of new Atlas Project
+      ClusterName:
+        default: Name of new cluster
+      ClusterRegion:
+        default: The AWS Region for Atlas Cluster
+      ClusterInstanceSize:
+        default: MongoDB Atlas Instance Size
+      DatabaseUserRoleDatabaseName:
+        default: MongoDB Atlas Database User Role Database Name
+      VPCCIDR:
+        default: VPC CIDR.
+      PrivateSubnetCIDR:
+        default: Private subnet CIDR
+      AvailabilityZone:
+         deafault: AZ fo Subnet
+      VPCRegion:
+        default: The AWS Region for VPC private endpoint.
+Parameters:
+  PublicKey:
+    Description: "Your MongoDB Cloud Public API Key"
+    Type: String
+    Default: "PublicKey"
+  PrivateKey:
+    Description: "Your MongoDB Cloud Private API Key"
+    Type: String
+    Default: "PrivateKey"
+  OrgId:
+    Description: "Your MongoDB Cloud Organization Id"
+    Type: String
+    Default: "OrgId"
+  ProjectName:
+    Description: "The name of the project."
+    Type: String
+    Default: "aws-quickstart"
+  ClusterName:
+    Description: Name of the cluster as it appears in Atlas. Once the cluster is created,
+      its name cannot be changed.
+    Type: String
+    Default: "Cluster-1"
+  ClusterInstanceSize:
+    Default: "M10" 
+    Description: Atlas provides different cluster tiers, each with a default storage capacity and RAM size. The cluster you select is used for all the data-bearing hosts in your cluster tier. See https://docs.atlas.mongodb.com/reference/amazon-aws/#amazon-aws.
+    Type: String
+    AllowedValues:
+    - "M10"
+    - "M20"
+    - "M30"
+    - "M40"
+    - "R40"
+    - "M40_NVME"
+    - "M50"
+    - "R50"
+    - "M50_NVME"
+    - "M60"
+    - "R60"
+    - "M60_NVME"
+    - "M80"
+    - "R80"
+    - "M80_NVME"
+    - "M100"
+    - "M140"
+    - "M200"
+    - "R200"
+    - "M200_NVME"
+    - "M300"
+    - "R300"
+    - "R400"
+    - "M400_NVME"
+    - "R700"
+  ClusterRegion:
+    Default: US_EAST_1
+    Description: AWS Region where the Atlas database runs.
+    Type: String
+    AllowedValues:
+      - "US_EAST_1"
+      - "US_EAST_2"
+      - "CA_CENTRAL_1"
+      - "US_WEST_1"
+      - "US_WEST_2"
+      - "SA_EAST_1"
+      - "AP_SOUTH_1"
+      - "AP_EAST_2"
+      - "AP_SOUTHEAST_1"
+      - "AP_SOUTHEAST_2"
+      - "AP_NORTHEAST_1"
+      - "AP_NORTHEAST_2"
+      - "EU_CENTRAL_1"
+      - "EU_WEST_1"
+      - "EU_NORTH_1"
+      - "EU_WEST_2"
+      - "EU_WEST_3"
+      - "EU_SOUTH_1"
+      - "ME_SOUTH_1"
+      - "AF_SOUTH_1"
+  ClusterMongoDBMajorVersion:
+    Description: The version of MongoDB
+    Type: String
+    Default: "4.4"
```

Comment:
Can't we default to the latest version and removed old version that customers should no longer use from the `AllowedValues` list?